src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementProjectService.java

```diff
@@ -253,6 +260,9 @@ public void updateProcurementProjectName(final Integer projectId, final String p
     combinedIds.addAll(teamIds);
     combinedIds.addAll(emailRecipientIds);
 
+    //update user
+    updateProjectUserMapping(dbProject, teamIds,principal);
```

Comment:
Might want to use the id results from the line below - i.e. `return combinedIds.stream()
        .map(i -> getTeamMember(i, teamIds, emailRecipientIds, projectOwner.getId()))`
        
Notice that after creating a project there are more values in the mapping table than are returned by the `/users` endpoint - that's because the `getTeamMember` function will strip out any people in Jaggaer that aren't in Conclave (like John Girling test users etc). Could potentially be a bit confusing if the API response doesn't match the table contents. And there would be no way to delete those users from the mapping table (when delete is implemented).